src/benchmark/tpch_benchmark.cpp

```diff
@@ -165,6 +169,33 @@ class TpchBenchmark final {
     } else {
       _create_report(std::cout);
     }
+
+    /**
+     * Visualize query plans
+     */
+    if (_enable_visualization) {
+      for (const auto tpch_idx_and_plans : _query_plans) {
```

Comment:
For now this is fine, but do you have any plans on how to deal with this once we have random parameters and thus potentially structurally different query plans?

Reply:
Don't have a solution for that problem, I'd tackle it when it emerges.